docs/spec/v1beta1/imagerepositories.md

```diff
@@ -190,7 +221,7 @@ type ImageRepositoryStatus struct {
 	// +optional
 	ObservedGeneration int64 `json:"observedGeneration,omitempty"`
 
-	// CannonicalName is the name of the image repository with all the
+	// CanonicalImageName is the name of the image repository with all the
```

Comment:
This section is taken from https://github.com/fluxcd/image-reflector-controller/blob/main/api/v1beta1/imagerepository_types.go#L99 which also has this incorrect name. Can you update that as well and run `make manifests` to generate the same in the CRD spec in `config/crd/bases/image.toolkit.fluxcd.io_imagerepositories.yaml`?

Reply:
It looks like this has been taken care of. I found another CannonicalName in the docs and updated it there 👍